src/plugins/data/public/query/mocks.ts

```diff
@@ -17,21 +17,24 @@
  * under the License.
  */
 
-import { QueryService, QueryStart, QuerySetup } from '.';
+import { QueryService } from '.';
+import { timefilterServiceMock } from './timefilter/timefilter_service.mock';
 
 type QueryServiceClientContract = PublicMethodsOf<QueryService>;
 
 const createSetupContractMock = () => {
-  const setupContract: jest.Mocked<QuerySetup> = {
```

Comment:
Why do we remove these return types? Same as in previous comment, it seems we are losing some type safety with this change:

![image](https://user-images.githubusercontent.com/9773803/67938450-cdbc9100-fbcf-11e9-8e91-52904884edbb.png)

Reply:
Managed to revert this